blueocean-dashboard/src/main/js/util/logDisplayHelper.js

```diff
@@ -31,6 +31,9 @@ export const getNodesInformation = (nodes) => {
         const isFailingNode = errorNodes.indexOf(item.id) > -1;
         const isRunningNode = runningNodes.indexOf(item.id) > -1;
 
+        const hasLogs = item.actions ? item.actions
+                .filter(action => action._class === 'org.jenkinsci.plugins.workflow.support.actions.LogActionImpl').length > 0
```

Comment:
I agree with you @michaelneale I said that to @tfennelly yesterday. I followed https://issues.jenkins-ci.org/browse/JENKINS-36170 but I will search for an example of the capabilities.